docs/UserGuide.md

```diff
@@ -409,43 +398,38 @@ Note:
 
 Shows only the specified columns from the *current* table view.
 
-If your screen is too cluttered, you may use this command to show only the columns you need!
-Take note that:
+[comment]: <> (If your screen is too cluttered, you may use this command to show only the columns you need!)
+
+Note:
 
 1. You can only use `showonly` on existing columns in the current table view, and
-2. The `showonly` command does not modify the list of residents being displayed. 
-   
-Filtered residents *stay filtered*!
+2. The `showonly` command does not modify the list of residents being displayed. [Filtered residents](#filtering-residents-by-field--filter) *stay filtered*.
 
 Format: `showonly LETTER [MORE_LETTERS]`
 
-Calling `filter /all g/M` (click [here](#filtering-residents-by-field--filter) to learn about `filter`) followed by
-`showonly n g h` will produce the following result:
+* Valid inputs include `i n p e r g h m t` (case-insensitive), which correspond to the first letter of each field in the table.
+    * This *should not* be confused with the `n/` or `p/` prefixes used in `add` or `filter`.
+* Letters *must* be separated by a single whitespace.
+* The order of each letter does not matter.
+* Duplicate letters are ignored.
+* There needs to be at least one column shown in the table at all times.
+* You can always use `reset` to restore the full table view!
```

Comment:
Good points to bring up! Improves clarity of this section of the UG quite a bit :D